helm/templates/deploy-additional-mysql.yml

```diff
@@ -0,0 +1,53 @@
+{{ if .Values.mysql.enabled }}
```

Comment:
remove that if mysql is always used/required?

Reply:
Yes, probably right. My intention was to allow an external MySQL database, but then I would have to make the hostname and port configurable too. I remove it for the moment.